apps/camera_pipe fails to build

```
$ CC="/Users/aam/gcc-4.6.2/bin/gcc" CXX="/Users/aam/gcc-4.6.2/bin/g++" make  out.png
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C ../../ bin/libHalide.a
make[1]: `bin/libHalide.a' is up to date.
/Users/aam/gcc-4.6.2/bin/g++ camera_pipe.cpp -I ../../include/ -g ../../bin/libHalide.a -o camera_pipe -ldl -lpthread -lz 
camera_pipe.cpp: In function ‘Halide::Func demosaic(Halide::Func)’:
camera_pipe.cpp:72:47: error: call of overloaded ‘absd(Halide::FuncRefExpr, Halide::FuncRefVar)’ is ambiguous
     Expr gvd_r = absd(g_gb(x, y-1), g_gb(x, y));
                                               ^
camera_pipe.cpp:72:47: note: candidates are:
camera_pipe.cpp:46:6: note: Halide::Expr absd(Halide::Expr, Halide::Expr)
 Expr absd(Expr a, Expr b) {
      ^
In file included from camera_pipe.cpp:1:0:
../../include/Halide.h:3176:13: note: Halide::Expr Halide::absd(Halide::Expr, Halide::Expr)
 inline Expr absd(Expr a, Expr b) {
             ^
```

--- a/apps/camera_pipe/camera_pipe.cpp
+++ b/apps/camera_pipe/camera_pipe.cpp
@@ -43,10 +43,6 @@ Func deinterleave(Func raw) {
     return deinterleaved;
 }
 
-Expr absd(Expr a, Expr b) {
-    return select(a > b, a - b, b - a);
-}
-
 Func demosaic(Func deinterleaved) {
     // These are the values we already know from the input
     // x_y = the value of channel x at a site in the input of channel y

